test/jdk/javax/swing/JTableHeader/TableHeaderBorderPositionTest.java

```diff
@@ -65,64 +64,40 @@ private static void test() {
         Dimension size = header.getPreferredSize();
         header.setSize(size);
 
-        int w = (int)Math.ceil(SCALE * size.width);
-        int h = (int)Math.ceil(SCALE * size.height);
-
-        BufferedImage imgHeader = new BufferedImage(w, h,
+        BufferedImage bufferedImage = new BufferedImage(WIDTH, HEIGHT,
                 BufferedImage.TYPE_INT_RGB);
-        Graphics2D g2d = imgHeader.createGraphics();
+        Graphics2D g2d = bufferedImage.createGraphics();
         g2d.scale(SCALE, SCALE);
-        try {
-            header.paint(g2d);
-        } finally {
-            g2d.dispose();
-        }
 
-        w = (int)Math.ceil(SCALE * WIDTH);
-        h = (int)Math.ceil(SCALE * HEIGHT);
-        BufferedImage imgData = new BufferedImage(w, h,
-                BufferedImage.TYPE_INT_RGB);
-        g2d = imgData.createGraphics();
-        g2d.scale(SCALE, SCALE);
         try {
+            header.paint(g2d);
+            g2d.translate(0, header.getHeight());
             table.paint(g2d);
         } finally {
             g2d.dispose();
         }
 
-        int verticalLineCol = (int)(table.getTableHeader().
-                getColumnModel().getColumn(0).getWidth() * SCALE) - 2;
-        int expectedRGB = imgData.getRGB(verticalLineCol, 0);
+        int verticalLineCol = (int) (table.getTableHeader()
+                                          .getColumnModel()
+                                          .getColumn(0)
+                                          .getWidth() * SCALE) - 2;
```

Comment:
```suggestion
        int verticalLineCol = (int)(table.getTableHeader()
                                         .getColumnModel()
                                         .getColumn(0)
                                         .getWidth() * SCALE) - 2;
```
If you add a space after the cast operator, add it everywhere; if you don't, add it nowhere.

I could've given you a suggestion based on my formatting, and I have the space after the cast operator because Java Coding Style recommends adding it. Yet there's no consensus about it in client-libs.